tests/functional/test_patient_access.py

```diff
@@ -5,32 +5,69 @@
 
 from tests.functional.utils.apigee_api import ApigeeDebugApi
 from tests.functional.utils.helper import (
-    generate_random_email_address,
-    get_add_telecom_email_patch_body,
+    generate_random_phone_number,
+    get_add_telecom_phone_patch_body,
 )
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+AUTH_PATIENT = {
+    "access": "patient",
+    "level": "P9",
+    "login_form": {"username": "9912003071"},
+}
+
+AUTH_PATIENT_P5 = {
+    "access": "patient",
+    "level": "P5",
+    "login_form": {"username": "9912003071"},
+}
+
+AUTH_PATIENT_p5 = {
+    "access": "patient",
+    "level": "p5",
+    "login_form": {"username": "9912003071"},
+}
+
+AUTH_PATIENT_P0 = {
+    "access": "patient",
+    "level": "P0",
+    "login_form": {"username": "9912003071"},
+}
+
+TEST_PATIENT_ID = "9912003071"
```

Comment:
Is there a reason you have hard coded this?

Reply:
It used to be picked up from the config, linked to an environment variable which didn't have the expected value in internal-dev during the builds